src/routes/news/components/TunewsDetailsFooter.js

```diff
@@ -47,7 +47,7 @@ class TunewsDetailsFooter extends React.PureComponent<PropsType> {
 
     return (
       <Footer>
-        <StyledContainer>{t('eNewsNo')}: {eNewsNo}</StyledContainer>
+        <StyledContainer>E-news No: {eNewsNo}</StyledContainer>
```

Comment:
Ah okay I guess we were not quite clear: It should be removed entirely, sorry for the confusion. So this should be:
```suggestion
        <StyledContainer>{eNewsNo}</StyledContainer>
```